_posts/2023-07-25-trino-fest-2023-datto.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Starburst Galaxy: A romance of many architectures"
+author: "Benjamin Jeter, Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Datto.png
+show_pagenav: false
+---
+
+Let's cut straight to the chase with this lightning talk from Benjamin Jeter, a
+data architect, platform manager, and data engineer at Datto. For those that are
+not familiar with Datto, they are an American cybersecurity and data backup
+company who are the leading global provider of security and cloud-based software
+solutions purpose-built for Managed Service Providers (MSPs). In Benjamin's
+talk, he goes through some of the considerations and design goals of a reference
+architecture pattern that they use and why they chose to use Trino on Starburst
```

Comment:
You cant use Trino on Starburst Galaxy ... Trino is inside it..


chose to to use Starburst Galaxy.


And then in the next paragraph have a quick sentence about it being a SaaS with Trino inside?